Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Build Stage
```

Comment:
Please do not use git clone or `ldd`. Also, I think you can omit `AFL_SKIP_CPUFREQ`. `@@` is not understood outside of the Mayhemfile context.